src/mtcp/mtcp_restart.c

```diff
@@ -246,6 +247,14 @@ main(int argc, char *argv[], char **environ)
     mtcp_abort();
     restart_slow_path();
   } else {
+    // Set this environment variable to debug with GDB inside mtcp_restart.c;
```

Comment:
1.  Is this comment accurate?  I think the comment should say that you need also to `rm bin/mtcp_restart && make` to re-make `bin/mtcp_restart` to load at the new address.  How about the more precise comment (still in 80 char's):
   `// Set env. var. DMTCP_DEBUG_MTCP_RESTART for easy debugging of 'mtcp_restart'. `
2. OPTIONAL COMMENT TO ADD: 
   `// This assumes no address conflict of mtcp_restart (0x111000000) w/ ckpt image.`

Reply:
We don't need to recompile as the mtcp_restart binary is defaulting to this new address.